plugins/powerd/fu-plugin-powerd.c

```diff
@@ -61,10 +62,7 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 }
 
 gboolean
-fu_plugin_update_prepare (FuPlugin *plugin,
-			  FwupdInstallFlags flags,
-			  FuDevice *device,
-			  GError **error)
+fu_plugin_update_prepare (FuPlugin *plugin, FwupdInstallFlags flags, FuDevice *device, GError **error)
```

Comment:
that's 102 chars wide -- didn't we standardize on 80?

Reply:
This was left for avoiding huge changes to existent code... The piece responsible of this is the `PenaltyExcessCharacter: '1'` I'll remove now to go back to under 80

Reply:
done